components/calendar/src/cal/chinese.rs

```diff
@@ -141,11 +146,17 @@ pub trait Rules: Clone + core::fmt::Debug + crate::cal::scaffold::UnstableSealed
 
 /// The [`Rules`] used in China.
 ///
-/// This type agrees with the official data published by the
+/// # Precise definition and limits
+///
+/// This calendar is intended to represent the traditional Chinese lunar calendar as used
+/// officially in the People's Republic of China as of 2025. This takes a best-effort approach
```

Comment:
it's not the official calendar of the country

```suggestion
/// in the People's Republic of China as of 2025. This takes a best-effort approach
```